Branch-SDK/Branch-SDK/BranchServerInterface.m

```diff
@@ -70,6 +70,8 @@ - (void)registerOpen:(BOOL)debug {
     if (osVersion) [post setObject:osVersion forKey:@"os_version"];
     NSString *uriScheme = [BNCSystemObserver getURIScheme];
     if (uriScheme) [post setObject:uriScheme forKey:@"uri_scheme"];
+    NSNumber *updateState = [BNCSystemObserver getUpdateState];
+    if (updateState) [post setObject:updateState forKeyedSubscript:@"update"];
     [post setObject:[NSNumber numberWithBool:[BNCSystemObserver adTrackingSafe]] forKey:@"ad_tracking_enabled"];
```

Comment:
also wanted to get your guys thought on this. this adds the check to register open - I don't think I should be doing that after your guys' feedback?